python/ray/train/v2/tests/test_data_integration.py

```diff
@@ -293,6 +293,118 @@ def check_resource_limits(config):
     trainer.fit()
 
 
+def test_per_dataset_execution_options_single(ray_start_4_cpus):
```

Comment:
can you add a test showing the default behavior of excluding train worker resources still applies to each dataset?